### ISSUE TITLE
Embed for Tweets versus Blockquote (html/js) with Text?

### ISSUE THREAD

**Issue by chrisaldrich** [pfefferle/wordpress-indieweb-press-this#10]
I've noticed that the IndieWeb Press This bookmarklet uses something like [embed]https://twitter.com/ChrisAldrich/status/707313378268618752 [/embed] in the main body of the content rather than the Twitter html/js embed method 
(a la: &lt;blockquote class="twitter-tweet" data-lang="en">&lt;p lang="en" dir="ltr"> etc... )

I suspect that WordPress is doing some processing on the back end with the [embed] short code to fetch and display the twitter card and thus isn't technically saving the actual content of the tweet the way the blockquote embed method does.  In the event of a Twitter silo death, this could spell trouble for users losing the data.

Cross reference: http://techlifeweb.com/bookmarklets-for-indieweb-favorites-and-retweets/ and http://stream.boffosocko.com/2016/scott-i-love-the-concept-of-your-bookmarklets-and-how

**Comment by petermolnar** [comment]
You're right, that is a WordPress internal thing. There is a plugin to disable embeds at large: https://wordpress.org/plugins/disable-embeds/ 
This is probably not a good idea to have as a dependency, but it is a good source to see how it can be done.

Note: you can also add my solution of using the regular Press This to the cross-reference list: https://petermolnar.eu/press-this-indieweb/

**Comment by pfefferle** [comment]
I am not sure if the plugin is doing what @chrisaldrich needs. The plugin only disables the embeds (oEmbed) of WordPress posts implemented in WordPress 4.4.

**Comment by pfefferle** [comment]
@chrisaldrich WordPress is doing the embed automatically, so I think the short code stuff is correct. In the final output should be the correct code.

**Comment by pfefferle** [comment]
@chrisaldrich WordPress uses the oEmbed API by Twitter to retrieve the correct code. You can look at the sourcecode of your tweet for the oEmbed links. Here is an example: https://publish.twitter.com/oembed?url=https://twitter.com/ChrisAldrich/status/707313378268618752

**Comment by chrisaldrich** [comment]
<a href="https://github.com/pfefferle/wordpress-indieweb-press-this/issues/10#issuecomment-223255042" class="u-in-reply-to"></a>Mostly I'm just thinking about this from the IndieWeb "Own Your Own Data" perspective, so if I'm reading this all correctly, the way the plugin is handling things, it's relying on Twitter's long term existence to serve up the physical content of the tweet?

I would suggest that using the blockquote method with the associated .js may be preferable for IndieWeb users as it saves the content of the tweet as a blockquote with the associated metadata. At worst, if Twitter disappeared, then one would at least "own" the text of the original which would minimally fallback to display as a simple blockquote rather than the fancier embedded Twitter card.
